piq/ssim.py

```diff
@@ -186,9 +186,8 @@ def _ssim_per_channel(x: torch.Tensor, y: torch.Tensor, kernel: torch.Tensor,
     Returns:
         Full Value of Structural Similarity (SSIM) index.
     """
-    if x.size(-1) < kernel.size(-1) or x.size(-2) < kernel.size(-2):
-        raise ValueError(f'Kernel size can\'t be greater than actual input size. Input size: {x.size()}. '
-                         f'Kernel size: {kernel.size()}')
+    assert x.size(-1) >= kernel.size(-1) and x.size(-2) >= kernel.size(-2), \
+        f'Kernel size can\'t be greater than actual input size. Input size: {x.size()}. Kernel size: {kernel.size()}'
```

Comment:
I think this one should remain an exception. Disabling it with python -O makes no sense, the code will fail later anyway. What we want to do with rising this exception is to make it fail early and save compute time.